stan/math/opencl/kernel_generator/optional_broadcast.hpp

```diff
@@ -53,10 +53,9 @@ class optional_broadcast_
 
   /**
    * generates kernel code for this and nested expressions.
-   * @param[in,out] generated set of already generated operations
-   * @param ng name generator for this kernel
    * @param row_idx_name  row index variable name
    * @param col_idx_name  column index variable name
+   * @param var_name_arg name generator for this kernel
    * @return part of kernel with code for this and nested expressions
```

Comment:
This one was wrong in the original PR. Not sure why this passed there. @t4c1 is the correction right?